Changelog.md

```diff
@@ -19,6 +19,15 @@
 * Fix array: missing pointer touch could potentially cause use of freed memory.
 * Fix array: unnecessary additional allocation due to a bug
 
+### Notable Internal API Changes
+
+* `tapAsync` is moved from `Stream.Parallal` to `Stream.IsStream.Transform`. It
+  now takes a `Fold` intead of a reducing function.
+* `tapAsyncK` has been added in `Stream.IsStream.Transform` preserving the
+  original behaviour of `tapAsync`.
```

Comment:
`tapAsyncK` has been moved from `Streamly.Internal.Data.Stream.Parallel` and renamed, it was called `tapAsync` in the original module.